### PR TITLE
Fixes #35 : Encrypt/Openssl Unset 'compression' Difference on __construct

### LINKED ISSUE #35: Encrypt/Openssl Unset Difference
Zend Framework 2.4.6-2.5.3, Zend\Filter\Encrypt\Openssl.php
I believe the unset should read `unset($options['compression']);` , lines 90-93.

```
if (array_key_exists('compression', $options)) {
    $this->setCompression($options['compression']);
    unset($options['compress']);
}
```

The code above I believe results in 'Undefined index: compression' later in code.  Current workaround is to use `...->setCompression($string)` .

### PR DESCRIPTION
**Pull request by samsonasik**
Fixes #35

### CODE REVIEW

**Comment by samsonasik** [comment]
ping @ezimuel

File: test/Encrypt/OpensslTest.php [hunk 1]
```diff
@@ -278,4 +279,15 @@ public function testEncryptionWithDecryptionAndCompressionWithPackagedKeys()
         $input = $filter->decrypt($output);
         $this->assertEquals('teststring', trim($input));
     }
+
+    public function testPassCompressionConfigWillBeUnsetCorrectly()
+    {
+        $filter = new OpensslEncryption([
+            'compression' => 'bz2'
```
Comment (webimpress) [inline test/Encrypt/OpensslTest.php]: trailing comma please

Comment (samsonasik) [inline test/Encrypt/OpensslTest.php]: fixed.

**Comment by samsonasik** [comment]
ping @ezimuel @weierophinney

**Comment by samsonasik** [comment]
ping @ezimuel is there a chance to merge it ?

**Comment by samsonasik** [comment]
ping @ezimuel @weierophinney

**Comment by samsonasik** [comment]
any other thing I can do to get it merged ? Thank you.

**Comment by weierophinney** [comment]
Thanks, @samsonasik

**Pull request merged**